db/migrate/20221121095445_rename_participant_declaration_outcomes.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class RenameParticipantDeclarationOutcomes < ActiveRecord::Migration[6.1]
+  def change
+    safety_assured do
+      rename_table :participant_declaration_outcomes, :participant_outcomes
```

Comment:
Personal preference/nit: since this is a new table and the first time we're introducing it prefer to only have the original migration with the correct name rather than a rename, but reckon you'll have to close/open the PR as this will upset the review app

Reply:
I would prefer that too but the rebase in this case would be horrible